format/spec.md

```diff
@@ -367,37 +367,35 @@ Iceberg tables must not use field ids greater than 2147483447 (`Integer.MAX_VALU
 
 The set of metadata columns is:
 
-| Field id, name                   | Type          | Description                                                                                            |
-|----------------------------------|---------------|--------------------------------------------------------------------------------------------------------|
-| **`2147483646  _file`**          | `string`      | Path of the file in which a row is stored                                                              |
-| **`2147483645  _pos`**           | `long`        | Ordinal position of a row in the source data file, starting at `0`                                     |
-| **`2147483644  _deleted`**       | `boolean`     | Whether the row has been deleted                                                                       |
-| **`2147483643  _spec_id`**       | `int`         | Spec ID used to track the file containing a row                                                        |
-| **`2147483642  _partition`**     | `struct`      | Partition to which a row belongs                                                                       |
-| **`2147483546  file_path`**      | `string`      | Path of a file, used in position-based delete files                                                    |
-| **`2147483545  pos`**            | `long`        | Ordinal position of a row, used in position-based delete files                                         |
-| **`2147483544  row`**            | `struct<...>` | Deleted row values, used in position-based delete files                                                |
-| **`2147483543  _change_type`**                    | `string`      | The record type in the changelog (INSERT, DELETE, UPDATE_BEFORE, or UPDATE_AFTER)                           |
-| **`2147483542  _change_ordinal`**                 | `int`         | The order of the change                                                                                     |
-| **`2147483541  _commit_snapshot_id`**             | `long`        | The snapshot ID in which the change occured                                                                 |
-| **`2147483540  _row_id`**                         | `long`        | A unique long assigned when row-lineage is enabled, see [Row Lineage](#row-lineage)                                  |
-| **`2147483539  _last_updated_sequence_number`**   | `long`        | The sequence number which last updated this row when row-lineage is enabled, see [Row Lineage](#row-lineage)              |
+| Field id, name                   | Type          | Description                                                                       |
```

Comment:
Minor: I think it's better to not alter table formatting to avoid larger than necessary spec changes. That helps preserve history so we can go back to when something changed (other than whitespace). I've been trying to get people to use no trailing whitespace in the last column for new tables and to avoid adding/removing whitespace for existing tables.

Reply:
I'll see what I can do here.  Unfortunately the formatting is automatically updated by Intellij which is why we're seeing this so frequently.

Reply:
I was able to disable this in my intellij, but then you have to reapply your changes mostly by hand :/